services/gitea/9.3.0/defaults/cm.yaml

```diff
@@ -46,9 +56,11 @@ data:
         periodSeconds: 10
         successThreshold: 1
         failureThreshold: 10
-    statefulset:
+    deployment:
```

Comment:
In the previous version (v8.x), PVC was automatically managed by statefulset, while in the new version (v9.x), it is automatically handled by helmchart. It may make sense to add those properties for upgrading, but not for installation. This is because, during installation, no PVC is created automatically if `persistence.create=false`.

Another point that isn’t entirely clear is that, [according to the documentation](  https://docs.gitea.com/installation/upgrade-from-gitea#backup-for-downgrade), database compatibility is not assured during major upgrades.

Reply:
hmmm yeah sounds like we'll need to somehow handle setting those config values during upgrade but not during install 🤔.

Reply:
I think we can manually create a PVC object, filling in its name value with the `data-gitea-0` that is the value in gitea `v8.x` and older versions.

```yaml
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name:  data-gitea-0
  namespace: kommander
  annotations:
    helm.sh/resource-policy: keep
spec:
  accessModes:
    - ReadWriteOnce
  volumeMode: Filesystem
  resources:
    requests:
      storage: 10Gi
```

Reply:
that would work. It would be nice to have new installations use a `RWX` PV but not required necessarily (we need to tackle it at some point (https://d2iq.atlassian.net/browse/D2IQ-88808)

Reply:
I see, I would say lets tackle it in it's ticket! just in case to be safe :smile: